apps/fdsnws/test/fdsnwstest.py

```diff
@@ -22,25 +22,24 @@
 import requests
 
 if sys.version_info[0] < 3:
-    from Queue import Queue #pylint: disable=E0401
+    from Queue import Queue  # pylint: disable=E0401
 else:
     from queue import Queue
 
-from seiscomp.fdsnws.utils import py3bstr # pylint: disable=C0413
+from seiscomp.fdsnws.utils import py3bstr  # pylint: disable=C0413
 
 ###############################################################################
 class FDSNWSTest(object):
 
-    #--------------------------------------------------------------------------
+    # --------------------------------------------------------------------------
     def __init__(self, port=8080):
         self.port = port
-        self.url = 'http://localhost:{}/fdsnws'.format(self.port)
+        self.url = "http://localhost:{}/fdsnws".format(self.port)
```

Comment:
I would rather keep the single quotes here as they represent an unnecessary change. The same applies for all following quote changes.

Reply:
@gempa-stephan, reformatting wasn't on purpose, sorry. I'm going to fix this.

Reply:
Ok. The formatting issue should be fixed now. I had auto-formatting enabled ...